src/main/java/com/conveyal/r5/transit/TripSchedule.java

```diff
@@ -80,6 +80,8 @@
      */
     public int[] stopSequences;
 
+    private TripPattern tripPattern;
```

Comment:
Just discussed the proposed `Map<TripSchedule, TripPattern>` solution quickly with @t2gran`, and he made a good counterproposal: TripSchedule does not need to implement OTP2's TripScheduleInfo interface. Instead the speed test can include a new wrapper class which implements TripScheduleInfo, and wraps two references, one to a TripSchedule and one to the TripPattern that owns it. This is arguably cleaner because it eliminates all references to the new OTP2 package from existing R5 code.